### PR TITLE
[optimization] console service code optimization

### PR DESCRIPTION
**Pull request by VampireAchao**
## What changes were proposed in this pull request

Issue Number: close #xxx 

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)
 no

### CODE REVIEW

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/domain/RestResponse.java [hunk 1]
```diff
@@ -17,6 +17,8 @@
 
 package org.apache.streampark.console.base.domain;
 
+import org.jetbrains.annotations.NotNull;
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/domain/RestResponse.java]: this dependency (maven pom) is not imported in any pom, in other words, can it still be compiled in a non intellij idae?

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/CommonUtils.java [hunk 2]
```diff
@@ -93,7 +90,8 @@ public static Boolean isEmpty(Object... objs) {
       }
 
       if (obj instanceof Iterable) {
-        if (((Iterable<?>) obj).iterator() == null || !((Iterable<?>) obj).iterator().hasNext()) {
+        ((Iterable<?>) obj).iterator();
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/CommonUtils.java]: If obj is null, will an exception occur?

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/CommonUtils.java [hunk 3]
```diff
@@ -191,11 +189,11 @@ public static Float toFloat(Object val) {
     return toFloat(val, 0f);
   }
 
-  public static List arrayToList(Object source) {
+  public static List<?> arrayToList(Object source) {
     return Arrays.asList(ObjectUtils.toObjectArray(source));
   }
 
-  public static boolean contains(Iterator iterator, Object element) {
+  public static boolean contains(Iterator<Object> iterator, Object element) {
```
Comment (wolfboys) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/CommonUtils.java]: I don't think we need to change here

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/CommonUtils.java (hunk 2 above)
Comment (VampireAchao) [reply streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/CommonUtils.java]: it will not

**Pull request closed without merging**